src/components/patientdb.js

```diff
@@ -74,8 +96,23 @@ function PatientDB(props) {
       } else if (label === 'detecteddistrict') {
         const city = document.getElementById('city');
         // Hide box
-        if (value === '') city.style.display = 'none';
-        else city.style.display = 'inline';
+        if (value === '') {
+          if (!city.classList.contains('hidden')) {
```

Comment:
combine this to ```if (value === '' && !city.classList.contains('hidden'))``` then ___

Reply:
**If value is '' and city element is hidden, we don't want any transition or changes to happen.**
If both the conditions are combined, the combined condition will give false on above scenario thus, the next else if condition will trigger which we don't want.